abci/src/signatures.rs

```diff
@@ -33,18 +33,32 @@ const VOTE_EXTENSION_REQUEST_ID_PREFIX: &str = "dpevote";
 /// SignDigest returns message digest that should be provided directly to a
 /// signing/verification function (aka Sign ID)
 pub trait SignDigest {
+    #[deprecated = "replaced with sign_hash() to unify naming between core, platform and tenderdash"]
     fn sign_digest(
```

Comment:
both `sign_digest` and `sign_hash` are not the best names. I'm reading this as a function that signs a hash. 
Can we do signature_hash?